modules/dists/BlockDist.chpl

```diff
@@ -2265,24 +2284,24 @@ proc BlockArr.doiScan(op, dom) where (rank == 1) &&
 
 ////// Factory functions ////////////////////////////////////////////////////
 
-@deprecated(notes="'newBlockDom' is deprecated - please use 'Block.createDomain' instead")
+@deprecated(notes="'newBlockDom' is deprecated - please use 'blockDist.createDomain' instead")
 proc newBlockDom(dom: domain) {
-  return dom dmapped Block(dom);
+  return dom dmapped blockDist(dom);
 }
 
-@deprecated(notes="'newBlockArr' is deprecated - please use 'Block.createArray' instead")
+@deprecated(notes="'newBlockArr' is deprecated - please use 'blockDist.createArray' instead")
 proc newBlockArr(dom: domain, type eltType) {
   var D = newBlockDom(dom);
   var A: [D] eltType;
   return A;
 }
 
-@deprecated(notes="'newBlockDom' is deprecated - please use 'Block.createDomain' instead")
+@deprecated(notes="'newBlockDom' is deprecated - please use 'blockDist.createDomain' instead")
 proc newBlockDom(rng: range...) {
   return newBlockDom({(...rng)});
 }
 
-@deprecated(notes="'newBlockArr' is deprecated - please use 'Block.createArray' instead")
+@deprecated(notes="'newBlockArr' is deprecated - please use 'blockDist.createArray' instead")
```

Comment:
Starting into this, and the next comment, I realized that it isn't quite so easy since we don't use these declarations to generate chpldocs, but document them all manually in the module's monolithic documentation.  That means that the hyperlinks are not so useful for the documentation, so we dropped this idea on the floor.  That said, I filed https://github.com/chapel-lang/chapel/issues/23290 as a future TODO to lean on chpldoc more within this module.